tasks/noci_test.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+EXIT_STATE=0
+
+# tests that aren't run on CI
+
+# jasmine specs with @noCI tag
+npm run citest-jasmine --  tests/*_test.js --tags noCI || EXIT_STATE=$?
```

Comment:
The link :arrow_up: is broken.

See usage here: https://github.com/plotly/plotly.js/commit/b42f021e691f14c5a8448a83e9ee5b704f5e805e